examples/image_classifier/resnet_152_batch/README.md

```diff
@@ -3,7 +3,7 @@ Run the commands given in following steps from the parent directory of the root
 
 ```bash
```

Comment:
remove this as handler has been removed

Reply:
Sorry, I didn't quite understand that. The readme has been updated to remove the custom handler. Did you mean to remove the whole example?

Reply:
Please ignore. Looks good to me.